Steg2

wleiding opened the pull request.

wleiding committed:
steg 2 first

första commit, ej fungerade checkboxes

wleiding committed:
steg2 second

Fungerande checkboxes, ej fungerande remove funktion

joelharrysson reviewed (changes requested):
Tycker det är snyggt att du la till en alertDialog för att lägga till en todo istället för att skapa en ny sida. Det går dock inte att ta bort saker från listan och som jag ser det så går det inte heller att filtrera listan? 
Lite tips: Ha en klass per fil + kommentera också gärna din kod. Det blir enklare att förstå den, både för sig själv och andra, när man beskriver vad de olika delarna ska göra. 

Fixa till detta så kan du gå vidare till nästa steg :)

wleiding committed:
Steg2 ver3

Steg2 ver3
-börjat med providers
-delat upp i olika filer
-filter finns visuellt, ingen funktion
nästa steg deletefunktion samt filter

wleiding committed:
steg2.4

Skapat en widget för deletefunktion, som inte fungerar. Det mesta behöver skrivas om inför api.
